cmd/buffer-api/main.go

```diff
@@ -25,11 +26,18 @@ func main() {
 	httpPortF := flag.String("http-port", "8000", "HTTP port")
 	debugF := flag.Bool("debug", false, "Enable debug log level.")
 	debugHTTPF := flag.Bool("debug-http", false, "Log HTTP client request and response bodies.")
+	cpuProf := flag.String("cpuprofile", "", "write cpu profile to `file`")
 	flag.Parse()
 
 	// Create logger.
 	logger := log.NewServiceLogger(os.Stderr, *debugF).AddPrefix("[bufferApi]")
 
+	// Start CPU profiling, if enabled
+	if filePath := *cpuProf; filePath != "" {
+		stop := cpuprofile.Start(filePath, logger)
+		defer stop()
+	}
+
```

Comment:
Uz viac krat som potreboval CPU profiler pre lokalny vyvoj, vyzdy som to potom zmazal.
Davam to teda na flag.